### PR TITLE
Fix For #1866 Display module by alphabetical order

### PR DESCRIPTION
**Pull request by salesagility-davidthomson**
Change text in all modules menu to make the task of translation easier.

## Description
see #1866

## Motivation and Context
It makes it easier for maintaining translations into different languages

## How To Test This
see #1866

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [x] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [ ] My change requires a change to the documentation.
- [x] I have read the [**How to Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM) guidelines.

### CODE REVIEW

File: include/language/en_us.lang.php [hunk 1]
```diff
@@ -92,10 +92,10 @@
             'Releases' => 'Releases',
             'Prospects' => 'Targets',
             'Queues' => 'Queues',
-            'EmailMarketing' => 'Email Marketing',
+            'EmailTemplates' => 'Email - Templates',
             'EmailTemplates' => 'Email Templates',
```
Comment (horus68) [inline include/language/en_us.lang.php]: EmailTemplates is now a duplicated key
```
-            'EmailMarketing' => 'Email Marketing',
 +            'EmailTemplates' => 'Email - Templates',
              'EmailTemplates' => 'Email Templates',
```

Comment (salesagility-davidthomson) [reply include/language/en_us.lang.php]: sorry about that I'll fix it straight away

Comment (salesagility-davidthomson) [reply include/language/en_us.lang.php]: thanks for letting me know

Comment (horus68) [reply include/language/en_us.lang.php]: And the EmailMarketing key, is to be removed?

Comment (salesagility-davidthomson) [reply include/language/en_us.lang.php]: I've added it back, I did the changes manually and missed those. Thanks for looking out for this.

**Comment by Dillon-Brown** [comment]
Assessed

**Pull request merged**